src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,51 @@
+.clock {
+  position: absolute;
+  @include round ($clock-size);
+  background-color: $clock-color;
+
+  &__center {
+    position: absolute;
+    z-index: 1;
+    top: 50%;
+    left: 50%;
+    @include round (50px);
+    transform: translate(-50%, -50%);
+
+    background-color: $color-central-circle;
+  }
+
+  &__big {
+    position: absolute;
+    display: flex;
+    top: 50%;
+    left: $left-placement-of-hands;
+    width: $hands-width;
+    height: $big-hand-height;
+
+    background-color: $color-big-hand;
+    animation: move 60s linear infinite;
+    transform-origin: top;
+  }
+
+  &__small {
+    position: absolute;
+    display: flex;
+    top: 50%;
+    left: $left-placement-of-hands;
+    width: $hands-width;
+    height: $small-hand-height;
+
+    background-color: $color-small-hand;
+    animation: move 3600s linear infinite;
+    transform-origin: top;
+  }
+
+  @keyframes move {
+    from {
+      transform: rotate(180deg);
+    }
+    to {
+      transform: rotate(540deg);
+    }
+  }
```

Comment:
Strange, so strange. Why not 0 --> 360?
The reason for this is that you have set `transform-origin: top;`. Set it to the `bottom` and use the `calc` function to set the proper value to the `top` property.